plugin.video.arteplussept/resources/lib/mapper.py

```diff
@@ -0,0 +1,182 @@
+from addon import plugin
+
+import hof
+import utils
+
+
+def map_categories_item(item):
+    return {
+        'label': utils.colorize(item.get('title'), item.get('color')),
+        'path': plugin.url_for('category', category_code=item.get('code'))
+    }
+
+
+def create_creative_item():
+    return {
+        'label': 'Creative I18N',
```

Comment:
Might not be the best place to report this issue, but shouldn’t this be localized somehow?

I always found it strange to have “Foobar I18N” showing in the UI.

Reply:
Yeah sorry about that, some unfinished features that stayed around. I'll clean it up.

Reply:
Done